packages/shared/package.json

```diff
@@ -2,6 +2,9 @@
   "name": "@aragonone/court-backend-shared",
   "version": "0.2.18",
   "private": true,
+  "workspaces": {
+    "nohoist": ["@aragon/court"]
```

Comment:
we should add the other ones I pointed out, clearly we will need them even tho those are not being used in the tests

Reply:
added